src/acom_music_box/data_output.py

```diff
@@ -64,12 +57,12 @@ def __init__(self, df, args):
             - output_format : str, optional
                 Format of the output file, either 'csv' or 'netcdf'. Defaults to 'csv'.
         """
-        self.df = df
+        self.df = df.copy(deep=True)
```

Comment:
is there reason for a Deep copy?

Reply:
Yes, without the deep copy, the changes made on the dataframe inside this class are reflected elsewhere, and that's not what we want